core/include/ldml/keyboardprocessor_ldml.h

```diff
@@ -0,0 +1,18 @@
+//
+// Generated File - do not edit
+//
+// This file is generated by core/tools/ldml-const-builder/build.sh
+// based on core/include/ldml/keyboardprocessor_ldml.ts
+//
+
+#pragma once
+#define LDML_META_SETTINGS_FALLBACK_OMIT 1
+#define LDML_META_SETTINGS_TRANSFORMFAILURE_OMIT 2
+#define LDML_META_SETTINGS_TRANSFORMPARTIAL_HIDE 4
+#define LDML_SECTION_KEYS ((uint32_t)'keys')
+#define LDML_SECTION_LOCA ((uint32_t)'loca')
+#define LDML_SECTION_META ((uint32_t)'meta')
+#define LDML_SECTION_SECT ((uint32_t)'sect')
+#define LDML_SECTION_STRS ((uint32_t)'strs')
+#define LDML_STRS_FLAGS_EXTEND 1
+#define LDML_VERSION 1.0
```

Comment:
I think we need to treat this as a string because we may end up with `"1.0.1"` in the future

Reply:
good idea.